modular_skyrat/modules/reagent_forging/code/forge_items.dm

```diff
@@ -94,45 +119,102 @@
 	icon_state = "hot_hammerhead"
 	spawn_item = /obj/item/forging/complete/hammer
 
-/obj/item/forging/incomplete/plate
-	name = "incomplete plate"
-	icon_state = "hot_plate"
-	average_hits = 10
-	average_wait = 0.5 SECONDS
-	spawn_item = /obj/item/forging/complete/plate
+/obj/item/forging/incomplete/pickaxe
+	name = "incomplete pickaxe head"
+	icon_state = "hot_pickaxehead"
+	spawn_item = /obj/item/forging/complete/pickaxe
+
+/obj/item/forging/incomplete/shovel
+	name = "incomplete shovel head"
+	icon_state = "hot_shovelhead"
+	spawn_item = /obj/item/forging/complete/shovel
 
 //"complete" pre-complete items
+/obj/item/forging/complete
```

Comment:
THIS is where the workbench thing was meant to be in the description. I recommend having it done based on examining this parent item, adding it as the usual on-examine `. += span_notice("You'll need a workbench to finish this!")`